### DISCUSSION TITLE
Does sagemaker support reading video input from s3 for BatchTransform jobs?

### DISCUSSION THREAD

**Discussion by velociraptor111** [aws/sagemaker-python-sdk#2362] [Help]
Thanks again for creating this great product. I have a couple of questions:

1) Can we read in video data as input to our batch transform jobs?
2) What is the limit to the processing time per-input for batch transform jobs? How can we increase this?
3) If 1) is currently not supported, how do you suggest we achieve this functionality? Is it a good idea to download the video from S3 ourselves and go on from there?

Related Question: 
https://github.com/aws/sagemaker-mxnet-serving-container/issues/77

Thanks

**Comment by nadiaya** [comment]
Thanks for using SageMaker! I've reached out to the relevant team.

**Comment by amitpeshwani** [comment]
Hi, @nadiaya any thoughts on using Batch transform for large video inference?
